frontend/src/Pages/ProgramOverviewDashboard.tsx

```diff
@@ -0,0 +1,303 @@
+            facility: {
+                name: 'MVCF'
+            },
+            facilitator: 'Linus Torvalds',
+            start_date: '2021-07-01',
+            end_date: '2021-07-31',
+            enrolled_residents: 10,
+            capacity: 15
+        },
+        {
+            id: 2,
+            program_id: 1,
+            facility_id: 1,
+            facility: {
+                name: 'BCF'
+            },
+            facilitator: 'Alex Trebek',
+            start_date: '2021-08-01',
+            end_date: '2021-08-31',
+            enrolled_residents: 14,
+            capacity: 30
+        },
+        {
+            id: 3,
+            program_id: 1,
+            facility_id: 1,
+            facility: {
+                name: 'Potosi'
+            },
+            facilitator: 'Mark Smith',
+            start_date: '2021-09-01',
+            end_date: '2021-09-30',
+            enrolled_residents: 16,
+            capacity: 40
+        }
+    ]
+};
+
+export default function ProgramOverview() {
+    const { id } = useParams<{ id: string }>();
+    const [page, setPage] = useState(1);
+    const [perPage, setPerPage] = useState(20);
+    const [selectedSections, setSelectedSections] = useState<number[]>([]);
+    const [searchTerm, setSearchTerm] = useState<string>('');
+    const [sortQuery, setSortQuery] = useState<string>('name_last asc');
+    const [includeActive, setIncludeActive] = useState(true);
+    const { data: programResp, error: programError } = useSWR<
+        ServerResponseOne<Program>,
+        AxiosError
+    >(`/api/programs/${id}`);
+    const program = programResp?.data;
+
+    const meta = {
+        current_page: page,
+        per_page: perPage,
+        total: sections.data.length,
+        last_page: Math.ceil(sections.data.length / perPage)
+    };
+
+    if (programError) {
+        <Error />;
+    }
+
+    const allSelected =
+        selectedSections.length === sections.data.length &&
+        sections.data.length > 0;
+
+    function handleToggleAll(checked: boolean) {
+        if (checked) {
+            setSelectedSections(sections.data.map((s) => s.id));
+        } else {
+            setSelectedSections([]);
+        }
+    }
+
+    function handleToggleRow(sectionId: number) {
+        setSelectedSections((prev) =>
+            prev.includes(sectionId)
+                ? prev.filter((id) => id !== sectionId)
+                : [...prev, sectionId]
+        );
+    }
+
+    const handleChange = (newSearch: string) => {
+        setSearchTerm(newSearch);
+        setPage(1);
+    };
+
+    const handleSetPerPage = (val: number) => {
+        setPerPage(val);
+        setPage(1);
+
+        setSortQuery(sortQuery); //Just to get eslint to stop complaining remove when this is is built out
+    };
+
+    return (
+        <div className="p-4">
+            <h1 className="text-2xl font-bold mb-2">{program?.name}</h1>
+            <p className="mb-4">{program?.description}</p>
```

Comment:
Can we use a lighter font-color, maybe one of the greys? And can we make the text smaller?